src/logic/safe/transactions/gasNew.ts

```diff
@@ -98,17 +112,18 @@ export const estimateSafeTxGas = async (safe, safeAddress, data, to, valueInWei,
 
     // 21000 - additional gas costs (e.g. base tx costs, transfer costs)
     const dataGasEstimation = estimateDataGasCosts(estimateData) + 21000
-
     const additionalGasBatches = [10000, 20000, 40000, 80000, 160000, 320000, 640000, 1280000, 2560000, 5120000]
 
     const batch = new web3.BatchRequest()
     const estimationRequests = additionalGasBatches.map(
       (additionalGas) =>
         new Promise((resolve) => {
+          // eslint-disable-next-line
+          // @ts-ignore
           const request = web3.eth.call.request(
```

Comment:
why are we adding those config comments?

If we're not using the `request` var, why are we instantiating it anyway?